packages/flutter/lib/src/widgets/notification_listener.dart

```diff
@@ -111,31 +130,21 @@ abstract class Notification {
 /// Notifications will trigger the [onNotification] callback only if their
 /// [runtimeType] is a subtype of `T`.
 ///
-/// To dispatch notifications, use the [Notification.dispatch] method.
-class NotificationListener<T extends Notification> extends StatelessWidget {
+/// To dispatch notifications, use the [BuildContext.dispatchNotification] method.
```

Comment:
Revert this since the old path is still valid now?

Reply:
done

Reply:
> done

This appears to be a lie 😝  (here and `scroll_activity.dart` as well).

Ultimately, of course it doesn't matter. But I also don't see a good argument to switch our recommendation.